tests/lightning_tensor/test_serialize_tensor.py

```diff
@@ -0,0 +1,936 @@
+        )
+        assert res[0] == s_expected
+
+    def test_sum(self, tn_backend, use_csingle, wires_map):
+        """Test that Sums are serialized correctly"""
+        tape = qml.tape.QuantumScript(
+            [],
+            [
+                qml.expval(
+                    qml.sum(
+                        0.5 * qml.prod(qml.PauliX(0), qml.PauliZ(1), qml.PauliX(2)),
+                        0.1 * qml.prod(qml.PauliZ(0), qml.Hadamard(2), qml.PauliY(1)),
+                    )
+                )
+            ],
+        )
+
+        hamiltonian_obs = get_module_name(tn_backend, "HamiltonianC", use_csingle)
+        tensor_prod_obs = get_module_name(tn_backend, "TensorProdObsC", use_csingle)
+        named_obs = get_module_name(tn_backend, "NamedObsC", use_csingle)
+
+        rtype = np.float32 if use_csingle else np.float64
+
+        tn_method = tn_backend
+        tn_method = "tn" if tn_method == "exact" else tn_method
+
+        res, _ = QuantumScriptSerializer(
+            device_name, use_csingle, tensor_backend=tn_method
+        ).serialize_observables(tape, wires_map)
+        assert len(res) == 1
+        assert isinstance(res[0], hamiltonian_obs)
+
+        coeffs = np.array([0.5, 0.1]).astype(rtype)
+        s_expected = hamiltonian_obs(
+            coeffs,
+            [
+                tensor_prod_obs(
+                    [named_obs("PauliX", [0]), named_obs("PauliZ", [1]), named_obs("PauliX", [2])]
+                ),
+                tensor_prod_obs(
+                    [named_obs("PauliZ", [0]), named_obs("PauliY", [1]), named_obs("Hadamard", [2])]
+                ),
+            ],
+        )
+        assert res[0] == s_expected
+
+    def test_multi_wire_identity(self, tn_backend, use_csingle, wires_map):
+        """Tests that multi-wire Identity does not fail serialization."""
+        tape = qml.tape.QuantumTape(measurements=[qml.expval(qml.Identity(wires=[1, 2]))])
+
+        tn_method = tn_backend
+        tn_method = "tn" if tn_method == "exact" else tn_method
+
+        res, _ = QuantumScriptSerializer(
+            device_name, use_csingle, tensor_backend=tn_method
+        ).serialize_observables(tape, wires_map)
+        assert len(res) == 1
+
+        named_obs = get_module_name(tn_backend, "NamedObsC", use_csingle)
+        assert res[0] == named_obs("Identity", [1])
+
+
+@pytest.mark.parametrize("wires_map", [wires_dict, None])
+@pytest.mark.parametrize("use_csingle", [True, False])
+class TestSerializeOps:
+    """Tests for the _ops function"""
+
+    wires_dict = {i: i for i in range(10)}
+
+    def test_basic_circuit(self, tn_backend, use_csingle, wires_map):
+        """Test expected serialization for a simple circuit"""
+        with qml.tape.QuantumTape() as tape:
+            qml.RX(0.4, wires=0)
+            qml.RY(0.6, wires=1)
+            qml.CNOT(wires=[0, 1])
+
+        tn_method = tn_backend
+        tn_method = "tn" if tn_method == "exact" else tn_method
```

Comment:
You can also make this a class attribute. This way you can avoid defining it every time.